utils/message/spoiler.go

```diff
@@ -0,0 +1,154 @@
+package message
+
+import (
+	"strings"
+)
+
+type spoilerToken struct {
+	tType spoilerTokenType
+	body  []rune
+}
+
+type spoilerTokenType int
+
+const (
+	spoilerTokenInvalid spoilerTokenType = iota
+	spoilerTokenExclamation
+	spoilerTokenContent
+	spoilerTokenSplit
+)
+
+func tokenizeSpoiler(msg string) []spoilerToken {
+	msgRunes := []rune(msg)
+	msgLen := len(msgRunes)
+	result := []spoilerToken{}
+	tokenStartIndex := 0
```

Comment:
こう読み替えたら理解できたんですが、こういうのはどうですか
```suggestion
	contentStartIndex := 0
```

Reply:
正しい…ってなったので書き換えました